vision/src/autogluon/vision/_gluoncv/object_detection.py

```diff
@@ -373,7 +357,7 @@ def fit(self, train_data, val_data=None, train_size=0.9, random_state=None, time
         self._logger.info(pprint.pformat(self._fit_summary, indent=2))
 
         if self._cleanup_disk:
-            shutil.rmtree(config['log_dir'], ignore_errors=True)
```

Comment:
is it possible that `log_dir` is passed as `/`? This is potentially very destructive operation if absolute paths are allowed.

Reply:
No because it is suffixed with a UUID str: 

```
log_dir = os.path.join(config.pop('log_dir', os.getcwd()), str(uuid.uuid4())[:8])
```